docs/UserGuide.md

```diff
@@ -300,13 +312,31 @@ Format: `view INDEX [-a]`
 
 * View the flashcard at the specified INDEX.
 * The index refers to the index number shown in the displayed flashcard list.
-* The index **must be a positive integer greater than 0** 1, 2, 3, …
+* The index **must be a positive integer greater than 0**. eg. 1, 2, 3, …
 *  If `-a` is specified, the answer and notes of the flashcard will be shown too.
 
 <div markdown="span" class="alert alert-primary">:memo: Note: Once another command is executed, the view pane will be returned to a blank state even if the shown
 flashcard was not modified/deleted.
 </div>
 
+### Viewing the statistics of flashcard `stats`
+
+View the statistics of a flashcard.
+
+Format: `stats INDEX`
```

Comment:
For the stats feature, add a similar disclaimer to the view feature that the viewpane will be returned to a blank state on new command executed.